esp/templates/program/modules/creditcardmodule_stripe/failure.html

```diff
@@ -40,15 +38,15 @@ <h3>Failure</h3>
 
 {% if error_type == "declined" %}
 <b>Credit card was declined.</b>  You may have triggered fraud detection, made a typo in the credit card information, or accidentally used an expired card.
-<ul><li>The payment processor reports: {{ error_info.message }}</li></ul>
+<center>The payment processor reports: {{ error_info.message }}</center>
 {% endif %}
 
 {% if error_type == "invalid" %}
-<b>Invalid request.</b>  This is usually caused by a duplicate request (e.g. "refreshing" your Web browser after paying).  It is possible that your payment went through successfully; please return to the payment page to check.
+<b>Invalid request.</b>  This is usually caused by a duplicate request (such as refreshing your web browser after paying).  It is possible that your payment went through successfully; <b>please return to the payment page to check</b>.
 {% endif %}
 
 {% if error_type == "auth" %}
-<b>Authentication error.</b>  The site was unable to authenticate with the payment processor; please let the administrators know so they can correct the issue.
+<b>Authentication error.</b>  The site was unable to authenticate with the payment processor; please <a href="mailto:tech@learningu.org">let the administrators know</a> so they can correct the issue.
```

Comment:
Do we definitely want this hard-coded?

Reply:
As opposed to contacting the group? I'm not sure they would know what to do with an auth error (which are rare as is)

Reply:
I'm not sure we ever want users contacting websupport directly, so I'd personally suggest putting the group's info here.

Reply:
this is tech@ not websupport, but I guess since this is a pre-payment error, there's no rush on our end, so yeah I can change it!